src/test/java/com/wikia/webdriver/testcases/userprofile/UserAvatar.java

```diff
@@ -59,20 +58,21 @@ public void clickOnAvatarRedirectsStaffUserToUserPage() {
 
   @Test(groups = "UserAvatar_staffUserCanRemoveAvatar", dependsOnMethods = "staffUserCanUploadAvatar")
   @Execute(asUser = User.STAFF)
-  @RelatedIssue(issueID = "MAIN-5960", comment =
-      "The Delete avatar button (and windows confirmation popup) " +
-      "have to be clicked twice in order to delete an avatar")
   public void staffUserCanRemoveAvatar() {
     UserProfilePageObject profile = new UserProfilePageObject(driver).openProfilePage(
         credentials.userNameStaff, wikiURL);
     String avatarUrl = profile.getAvatarImageSrc();
     profile.clickRemoveAvatar();
     profile.verifyAvatar();
 
-    profile.verifyAvatarChanged(avatarUrl);
-    String changedAvatarUrl = profile.getAvatarImageSrc();
-    profile.verifyAvatarVisible();
+    profile.openWikiPage(); //user needs to visit other page to get avatar refreshed
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ATrailingCommentCheck)